frontend/public/components/_nav.scss

```diff
@@ -112,7 +127,7 @@ $active-border: 3px;
 }
 
 .navigation-container__section__title__icon {
-  color: $color-os-nav-icon;
```

Comment:
`$color-os-nav-icon` is already set to `$color-pf-black-600`.  No change needed.